bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts

```diff
@@ -80,10 +78,21 @@ export class ContactComponent implements OnInit {
   budgets: any = ['Slide for budget amount'];
   timelines: any = ['Slide for time frame'];
   timeframe: any = [0, 35];
-  years: any = ['1980 and older', 1981, 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1989, 1990, 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020];
-foundations: any = ['New Business', 'Existing Business'];
+  years = [];
+  foundations: any = ['New Business', 'Existing Business'];
+
   ngOnInit() {
+    this.years = this.calculateYears(new Date().getFullYear(), 40);
+    this.years.push('Older than 1980');
+  }
 
+  public calculateYears(yearList, yearsSpan): any []{
```

Comment:
Still not right.  Look at previous comment.  yearList and yearSpan need to have a type declared.

Reply:
Added the declared types to the parameters, For the yearList it was only capable of accepting a number as a type due to the nature of the method "new Date.getFullYear".